cranelift/codegen/src/isa/aarch64/inst/args.rs

```diff
@@ -175,7 +179,7 @@ impl MemArg {
     }
 
     /// Memory reference using an address in a register and an offset, if possible.
-    pub fn reg_maybe_offset(reg: Reg, offset: i64, value_type: Type) -> Option<MemArg> {
+    pub fn reg_offset(reg: Reg, offset: i64, value_type: Type) -> Option<MemArg> {
```

Comment:
Could this return `MemArg::RegOffset` (not in an option anymore) unconditionally now?

Reply:
Ah, I had somehow stopped halfway through updating that, but it turns out it's dead code, so I just removed it (all the uses build a `MemArg` directly).